src/utils/functions.ts

```diff
@@ -0,0 +1,81 @@
+import classnames from 'classnames';
+import { Person } from '../types';
+import { YEARS_PER_CENTURY } from './constants';
+import { TypeSort } from '../types/TypeSort';
+
+export function getFatherPerson(persons: Person[], person: Person) {
+  return persons.find(({ name }: Person) => name === person.fatherName);
+}
+
+export function getMotherPerson(persons: Person[], person: Person) {
+  return persons.find(({ name }: Person) => name === person.motherName);
+}
+
+export const getLinkClass = ({ isActive }: { isActive: boolean }) => {
+  return classnames('navbar-item', {
+    'has-background-grey-lighter': isActive,
+  });
+};
+
+export const hasIncludeQuery = (name: string, query: string) => {
+  const normalizedQuery = query.toLowerCase();
+
+  return name.toLowerCase().includes(normalizedQuery);
+};
+
+export const getFilteredPeople = (people: Person[], search: string) => {
+  const params = new URLSearchParams(search);
+
+  const sex = params.get('sex') || '';
+  const centuries = params.getAll('centuries') || [];
+  const query = params.get('query') || '';
+  const sort = params.get('sort') || '';
+  const order = params.get('order') || '';
```

Comment:
Helper functions should know nothing about query parameters
Also, it's better to split your function into two, one for filtering and the second for sorting